itests/kit/ensemble_opts.go

```diff
@@ -31,7 +31,7 @@ type ensembleOpts struct {
 
 var DefaultEnsembleOpts = ensembleOpts{
 	pastOffset: 10000000 * time.Second, // time sufficiently in the past to trigger catch-up mining.
-	upgradeSchedule: stmgr.UpgradeSchedule{{
+	upgradeSchedule: stmgr.UpgradeSchedule{stmgr.Upgrade{
```

Comment:
Did this `stmgr.Upgrade` name change after the last rebase from `master`? I wasn´t aware of this

Reply:
That's just a type annotation for clear linting